Marlin/Configuration_adv.h

```diff
@@ -1434,24 +1434,26 @@
 
 #endif // HAS_DISPLAY || DWIN_LCD_PROUI
 
-// Add the G-code 'M73' to set / report the current job progress
+// Add the G-code 'M73' to set / report the current job progress. Until M73 P/R is sent display Marlin's built-in estimate.
```

Comment:
Maybe "display the current job progress" would be better wording so it's not confused with the `M73_REPORT  // Report M73 values to host` option below. Otherwise, it sounds like we have two `M73` "reporting" options.

Reply:
Oh, yep. It don't mix with first sentence.